cluster/userdata-master.yaml

```diff
@@ -757,6 +765,90 @@ write_files:
               - kubectl
               - proxy
 
+  - path: /srv/kubernetes/manifests/prometheus-node-exporter-de.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: DaemonSet
+      metadata:
+        name: prometheus-node-exporter
+        labels:
+          app: prometheus
+          component: node-exporter
+        namespace: kube-system
+      spec:
+        template:
+          metadata:
+            name: prometheus-node-exporter
+            labels:
+              app: prometheus
+              component: node-exporter
+          spec:
+            containers:
+            - image: prom/node-exporter:0.12.0
+              name: prometheus-node-exporter
+              ports:
+              - name: prom-node-exp
+                containerPort: 9100
+                hostPort: 9100
+            hostNetwork: true
+            hostPID: true
+  - path: /srv/kubernetes/manifests/prometheus-node-exporter-svc.yaml
+    content: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        annotations:
+          prometheus.io/scrape: 'true'
+        name: prometheus-node-exporter
+        labels:
+          app: prometheus
+          component: node-exporter
+      spec:
+        clusterIP: None
+        ports:
+          - name: prometheus-node-exporter
+            port: 9100
+            protocol: TCP
+        selector:
+          app: prometheus
+          component: node-exporter
+        type: ClusterIP
+
+  - path: /etc/kubernetes/nginx/nginx.conf
+    content: |
+      user nginx;
+      worker_processes 1;
+
+      error_log /dev/stdout info;
+
+      events {
+        worker_connections 1024;
+      }
+
+      http {
+        default_type application/octet-stream;
+
+        server_tokens off;
+        access_log off;
+
+        server {
+          listen 8082;
+          server_name _;
+
+          location / {
+            if ($request_method != GET) {
+              return 403;
+            }
+            proxy_pass_request_body off;
+            proxy_pass http://localhost:8080;
+          }
+
+          location ~ /secrets(/|$) {
```

Comment:
I would still rather have a whitelist instead of a blacklist, it's hard to catch (and maintain) everything with a blacklist